specifications/raml/api.raml

```diff
@@ -227,17 +145,8 @@ traits:
           200:
             body:
               application/json:
-                schema: |
-                  {
-                    "$schema": "http://json-schema.org/draft-03/schema",
-                    "type": "object",
-                    "properties": {
-                      "tracks": {
-                        "type": "array",
-                        "items": {"$ref": "track"}
-                      }
-                    }
-                  }
+                type: Track[]
```

Comment:
Would this take into account that the array is returned wrapped within a `tracks` JSON property? (same applies to some other endpoints like the one to fetch several tracks).